tests/ctst/steps/utils/utils.ts

```diff
@@ -197,17 +197,18 @@ async function createBucketWithConfiguration(
 
 async function putObject(world: Zenko, objectName?: string) {
     world.addToSaved('objectName', objectName || Utils.randomString());
-    const objectNameArray = world.getSaved<string[]>('objectNameArray') || [];
-    objectNameArray.push(world.getSaved<string>('objectName'));
-    world.addToSaved('objectNameArray', objectNameArray);
     await uploadSetup(world, 'PutObject');
     world.addCommandParameter({ key: world.getSaved<string>('objectName') });
     world.addCommandParameter({ bucket: world.getSaved<string>('bucketName') });
     const result = await S3.putObject(world.getCommandParameters());
+    const versionId = extractPropertyFromResults<string>(result, 'VersionId');
+    world.saveCreatedObject(world.getSaved<string>('objectName'), versionId || '');
```

Comment:
`objectName` is specified in parameter, should be the "source of truth" in this function...
(it is too easy to forget to read it from saved and use the variable directly... which may lead to issue when refactoring/...)

```suggestion
async function putObject(world: Zenko, objectName: string = Utils.randomString()) {
    world.addToSaved('objectName', objectName);
    await uploadSetup(world, 'PutObject');
    world.addCommandParameter({ key: objectName });
    world.addCommandParameter({ bucket: world.getSaved<string>('bucketName') });
    const result = await S3.putObject(world.getCommandParameters());
    const versionId = extractPropertyFromResults<string>(result, 'VersionId');
    world.saveCreatedObject(objectName, versionId || '');
```